orbit/pkg/table/windowsupdatetable/windowsupdate_test.go

```diff
@@ -14,6 +16,55 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type WindowsUpdatesSearcherMock struct {
```

Comment:
Nit, no need to export this type.

Reply:
Done in 2499c119bb5b86a4d838971449cdd87b4fa6fcf0